lib/core/network/models/transaction_model.dart

```diff
@@ -32,6 +32,10 @@ sealed class TransactionModel extends Equatable {
     final actor = act['authorization'].first['actor'];
     final transactionId = json['trx_id'];
 
+    if(account != 'dao.hypha') {
```

Comment:
husd.hypha but its must be dao.hypha

This check fails.

What should we do @n13

Reply:
there is also voice.hypha

Reply:
Fixed